File: base/src/main/java/de/hipp/pnp/base/converter/persistent/ListConverter.java
package de.hipp.pnp.base.converter.persistent;

import java.util.ArrayList;
import java.util.List;

import jakarta.persistence.Converter;
import org.springframework.stereotype.Component;

@Converter(autoApply = true)
@Component
public class ListConverter extends BaseConverter<List<Object>> {

    public ListConverter() {
        this.type = new ArrayList<Object>();
    }
}

File: base/src/main/kotlin/de/hipp/pnp/base/converter/persistent/ListConverter.kt
package de.hipp.pnp.base.converter.persistent

import jakarta.persistence.Converter
import org.springframework.stereotype.Component

@Converter(autoApply = true)
@Component
class ListConverter : BaseConverter<List<Any>>() {
    init {
        this.type = ArrayList<Any>()
    }
}
